### PR TITLE
lsattr: readed the page

### PR DESCRIPTION
**Pull request by Cvetomird91**
I am opening a new pull request for the lsattr command.

In the previous pull request my fork of the branch wasn't synced with the upstream master branch.

Previous PR: #502

### CODE REVIEW

**Comment by igorshubovych** [comment]
http://www.tutorialspoint.com/unix_commands/lsattr.htm

How about adding the simplest examples
`lsattr`
`lsattr {{/path/to/folder}}`

**Comment by Cvetomird91** [comment]
Added.

File: pages/linux/lsattr.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# lsattr
+
+> List file attributes on a Linux file system.
+
+- Display the attributes of the files in the current directory:
+
+`lsattr`
+
+- List the attributes of files in {{path}}:
```
Comment (igorshubovych) [inline pages/linux/lsattr.md]: We usually do not repeat tokens in description. It is probably better to say
- List the attributes of files in a given path:

**Comment by Cvetomird91** [comment]
The unnecessary token has been removed from the second description.

**Comment by rubenvereecken** [comment]
Looking good! Thanks for taking the time to recreate the PR and for a great contribution.

**Comment by Cvetomird91** [comment]
Thank you for your kind words! I'm glad to assist. I'm learning a lot while contributing to TLDR. :)

**Pull request merged**